src/renderer/src/Pages/ProjectDetails/Nodes/Collaborators/index.tsx

```diff
@@ -20,11 +20,11 @@ const Collaborators: React.FC<CollaboratorProps> = ({ project, showModal, setSho
     const [error, setError] = useState<string | null>(null);
 
     useEffect(() => {
-        if (project?.remote) {
+        if (project?.remote !== '') {
```

Comment:
if project is undefined or the remote is undefined then it will enter :c
```
    if (project && porject.remote !== undefined && project.remote !== '') { 
```